### ISSUE TITLE
Can't enable source control. needs to be initialized. UE (4.23.0)

### ISSUE THREAD

**Issue by Marenz** [SRombauts/UE4GitPlugin#124]
My problem: The initialization of the git project fails with
`Failed to enable Git source control. You need to initialize the project as a Git repository first.`

My steps:

1. Create new project (UE 4.23.0)
2. Add plugin
3. Confirm rebuild due to engine version differences
4. Open and fill out "Connect to SourceControl", checking all options available.
5. Click on accept settings
6. Receive error noted above.

![image](https://user-images.githubusercontent.com/424752/74550435-af498e00-4f51-11ea-8fb0-2d3922acdea0.png)

**Comment by Marenz** [comment]
Ah.. I need to click on initialize of course. At this point i am no longer sure what I clicked for real. 
Maybe the accept button should be disabled if no repo was found yet?

**Comment by SRombauts** [comment]
Yes I get this is not ideal, but changing this Accept button would need to change the Editor itself.
I'll have à look at it anyway since it would be interesting
